The module should also ignore PAM_REINITIALIZE_CRED to avoid spurious errors.

--- Linux-PAM-0.75/modules/pam_group/pam_group.c	Sun Nov 26 02:32:39 2000
+++ Linux-PAM-0.75/modules/pam_group/pam_group.c	Tue Jul 10 00:14:23 2001
@@ -764,7 +764,7 @@
     /* only interested in establishing credentials */
 
     setting = flags;
-    if (!(setting & PAM_ESTABLISH_CRED)) {
+    if (!(setting & (PAM_ESTABLISH_CRED | PAM_REINITIALIZE_CRED))) {
 	D(("ignoring call - not for establishing credentials"));
 	return PAM_SUCCESS;            /* don't fail because of this */
     }
